app/src/main/res/layout/fragment_video_detail.xml

```diff
@@ -365,49 +329,43 @@
                         <View
                             android:layout_width="match_parent"
                             android:layout_height="1px"
-                            android:layout_alignParentBottom="true"
                             android:layout_marginLeft="8dp"
                             android:layout_marginRight="8dp"
                             android:background="?attr/separatorColor"/>
 
-                    </RelativeLayout>
+                    </LinearLayout>
 
                     <!--NEXT AND RELATED VIDEOS-->
-                    <RelativeLayout
+                    <LinearLayout
                         android:id="@+id/detail_related_streams_root_layout"
                         android:layout_width="match_parent"
-                        android:layout_height="match_parent"
-                        android:layout_below="@+id/detail_description_root_layout"
-                        android:layout_centerHorizontal="true"
+                        android:layout_height="wrap_content"
                         android:layout_gravity="center_horizontal|bottom"
-                        android:layout_marginTop="14dp">
+                        android:layout_marginTop="14dp"
+                        android:orientation="vertical">
 
                         <TextView
                             android:id="@+id/detail_next_stream_title"
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:layout_alignParentLeft="true"
                             android:layout_marginLeft="12dp"
                             android:text="@string/next_video_title"
                             android:textAllCaps="true"
                             android:textAppearance="?android:attr/textAppearanceMedium"
                             android:textSize="@dimen/video_item_detail_next_text_size"
                             tools:ignore="RtlHardcoded"/>
 
-                        <LinearLayout
+                        <android.support.v7.widget.RecyclerView
```

Comment:
Oh! I didn't have a nested scroll view (maybe my phone's to old).
Maybe it could have been disabled with [setNestedScrollingEnabled(false)](https://developer.android.com/reference/android/support/v4/view/NestedScrollingChild.html#setNestedScrollingEnabled(boolean)).